pkg/imgpkg/image/dir_image.go

```diff
@@ -135,6 +135,11 @@ func (i *DirImage) extractTarEntry(header *tar.Header, input io.Reader) error {
 	userPermission := int64(mode & 0700)
 	permMode := os.FileMode(userPermission | userPermission>>3 | userPermission>>6)
 
+	// if group/all permission is set assume that the image was created to keep the full original permission set
```

Comment:
This comment doesn't really tell me why we need to do this.

> assume that the image was created to keep the full original permission set

What does that mean exactly?

Reply:
^ +1